src/main/java/teammates/ui/controller/ActionFactory.java

```diff
@@ -26,14 +26,7 @@
         map(INSTRUCTOR_FEEDBACK_RESULTS_PAGE, InstructorFeedbackResultsPageAction.class);
         map(INSTRUCTOR_FEEDBACK_RESULTS_DOWNLOAD, InstructorFeedbackResultsDownloadAction.class);
         map(INSTRUCTOR_STUDENT_RECORDS_AJAX_PAGE, InstructorStudentRecordsAjaxPageAction.class);
-
         map(STUDENT_FEEDBACK_RESULTS_PAGE, StudentFeedbackResultsPageAction.class);
-        map(STUDENT_PROFILE_PICTURE_UPLOAD, StudentProfilePictureUploadAction.class);
-        map(STUDENT_PROFILE_PICTURE_EDIT, StudentProfilePictureEditAction.class);
-        map(STUDENT_PROFILE_CREATEUPLOADFORMURL, StudentProfileCreateFormUrlAction.class);
-
-        map(CREATE_IMAGE_UPLOAD_URL, CreateImageUploadUrlAction.class);
-        map(IMAGE_UPLOAD, ImageUploadAction.class);
```

Comment:
You should remove the classes as well, not just the mapping. Also, if possible, all the above constants in `Const.ActionURIs`.

Reply:
Files are deleted before already, I'll remove the constants 👍